rust/storage/src/admissioncontrolleds3.rs

```diff
@@ -444,5 +450,7 @@ mod tests {
         test_multipart_get_for_size(1024 * 1024 * 7).await;
         // At > 8 MB.
         test_multipart_get_for_size(1024 * 1024 * 19).await;
+        // Greater than NAC limit i.e. > 5*8 MB = 40 MB.
```

Comment:
This looks like 100MB

Reply:
I think he's saying the NAC limit is 5*8, so this will be greater than that and trigger the bug we found? But I am not sure where 5 comes from

Reply:
Oh 5 comes from https://github.com/chroma-core/chroma/pull/2991/files#diff-1ed7b10fcd474390600189380273049accc9c2356624e8a4937e8e94e53cea7bR67